src/libvmemcache.h

```diff
@@ -143,6 +149,9 @@ int vmemcache_get_stat(VMEMcache *cache,
 	void *value,
 	size_t value_size);
 
+void vmemcache_config_set(VMEMcache *cache, enum vmemcache_config cfg,
+				size_t val);
```

Comment:
`..., void *val, size_t vsize)`
To make sure we can support strings/structures in the future.